### DISCUSSION TITLE
[3.3.7.dev1] User Profile "Number of Slots" Removed?

### DISCUSSION THREAD

**Discussion by Climbatiz** [nicotine-plus/nicotine-plus#3179] [Ideas]
wasn't sure where to ask this, but anyway, in latest Dev release if you goto a persons User Profile it nolonger displays how many Slots they have for uploads but rather just if they have "free" slots or not, why was this removed? i thought it was good info to have

**Comment by slook** [comment]
https://github.com/nicotine-plus/nicotine-plus/commit/3972846ef40b07cfaa90090bae081075d90734ed

**Comment by mathiascode** [comment]
"Queued uploads" and "Upload slots" are only displayed when no free slots are available, since they didn't seem that relevant if you can download files immediately. Is there a reason you always need to see this information?

**Comment by Climbatiz** [comment]
i'd like to see it to know how many Upload Slots a person has when i look at their profile to guage how much they will upload to users

**Reply by mathiascode** [reply]
I don't think the number of upload slots is a good representation for this, since the number of upload slots is always (number of occupied slots + 1) when using dynamic upload slots (default in Nicotine+).

With that being said, we could revert to always showing the number of slots. It's at least somewhat more useful than the number of queued uploads.

**Reply by mathiascode** [reply]
Changed a few things:
- Upload slots always visible
- Queued uploads only visible when there are queued uploads
- Moved user country next to the username

Does this work for you? The intention is to free up some vertical space for the description.

**Reply by Climbatiz** [reply]
> Changed a few things:
> Does this work for you? The intention is to free up some vertical space for the description.

yeah, that sounds fine ;}
